internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
+	Filename = "build.yml"
+	// CurrentVersion os the manifest file format. See Manifest.SDK.
+	CurrentVersion = "2"
+)
+
+// Artifact is a file copied from one path to another during the build.
+type Artifact struct {
+	Path string `yaml:"path"`
+	Dest string `yaml:"dest"`
+}
+
+// Manifest used for the declarative build system for drivers.
+type Manifest struct {
+	// SDK version. Used to track future changes to this file. Current version is '2'.
+	SDK string `yaml:"sdk"`
+
+	Native struct {
+		// Image is a Docker image used as the native driver runtime.
+		Image string `yaml:"image"`
+		// Static is a list of files that will be copied from native driver's source directory
+		// to the final driver image. Note that those files cannot be modified by the build.
```

Comment:
Does "cannot" in this context mean "we will report an error" or "bad things will silently happen"?

If it's the first, we should (separately) make sure we check for it.

If it's the second, I suggest `must not be modified`, to emphasize it's a responsibility of the build tool rather than a restraint of the manifest user.

Reply:
Neither, it's more like "it's impossible to do so". I should probably use this wording in the comment.